models/logs.js

```diff
@@ -0,0 +1,65 @@
+const firestore = require("../utils/firestore");
+const logsModel = firestore.collection("logs");
+const admin = require("firebase-admin");
+
+/**
+ * Adds log
+ *
+ * @param type { String }: Type of the log
+ * @param data { String }: Body of the log
+ */
+const add = async (type, data) => {
+  try {
+    const log = {
+      type,
```

Comment:
How are we going to query the logs specific to any entity whenever required? These logs seems like those cant be queried.